README.md

```diff
@@ -69,4 +39,4 @@ Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
 ## Contributing
 
 * Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
+* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
```

Comment:
Let's remove the line 41 and in line 42 talk about only examples.

Reply:
done